client/dashboard/app/components/Sidebar.tsx

```diff
@@ -48,6 +48,19 @@ const Sidebar = () => {
               <BiRocket />
             </Link>
           </li>
+          <li title="FAQ's">
+            <Link
+              to="https://recruitment.srmkzilla.net/faq"
```

Comment:
Instead of hardcoding, expose `LANDING_URL` to the client by including it in the `LoaderFunction` in `root.tsx`